cmd/kubeadm/app/apis/kubeadm/v1beta4/types.go

```diff
@@ -453,3 +453,36 @@ type Patches struct {
 	// +optional
 	Directory string `json:"directory,omitempty"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ResetConfiguration contains a list of elements that is specific "kubeadm reset"-only runtime information.
+type ResetConfiguration struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// SkipPhases is a list of phases to skip during command execution.
+	// The list of phases can be obtained with the "kubeadm reset phase --help" command.
+	// +optional
+	SkipPhases []string `json:"skipPhases,omitempty"`
+
+	// Force flag instructs kubeadm to reset the node without prompting for confirmation
+	// +optional
+	Force bool `json:"forceReset,omitempty"`
```

Comment:
Name is different with that `forceReset` in json. Not sure if this is OK.

Reply:
yes, this should be updated as well.